packages/bento-frontend/src/pages/caseDetail/caseDetailView.js

```diff
@@ -71,7 +70,7 @@ const CaseDetail = ({ data, filesOfSamples, classes }) => {
 
   const samplesData = data.samples.map((s) => {
     const files = filesOfSamplesObj[s.sample_id];
-    const sample = _.cloneDeep(s);
+    const sample = { ...s };
```

Comment:
@rana22 why not directly assign "s" to sample?

Reply:
updated